README.md

```diff
@@ -2,7 +2,11 @@
 [![Jenkins Plugin](https://img.shields.io/jenkins/plugin/v/pipeline-input-step.svg)](https://plugins.jenkins.io/pipeline-input-step)
 [![Jenkins Plugin Installs](https://img.shields.io/jenkins/plugin/i/pipeline-input-step.svg?color=blue)](https://plugins.jenkins.io/pipeline-input-step)
 
-Adds the Pipeline `input` step to wait for human input or approval.
+Adds the Pipeline `input` step to wait for human input or approval. 
+
+For further understanding,check the [documentation](https://plugins.jenkins.io) for pipeline-input-step plugin.  The  [Pipeline Syntax Snippet Generator](https://www.jenkins.io/redirect/pipeline-snippet-generator) guides the user to select checkout options.
```

Comment:
I think it would be better if the first sentence of the paragraph recommended the Pipeline Syntax Snippet Generator and the second sentence directed them to the detailed documentation of the input step.  Directing them to the https://plugins.jenkins.io page causes the reader to open a page with very little content.
```suggestion
Use the  [Pipeline Syntax Snippet Generator](https://www.jenkins.io/redirect/pipeline-snippet-generator) to select options for the `input` step.
For further understanding, check the [pipeline input step plugin documentation](https://www.jenkins.io/doc/pipeline/steps/pipeline-input-step/).
```